### ISSUE TITLE
Feature Request: A possibility to write text when landscape mode is active

### ISSUE THREAD

**Issue by DariusMofakhami** [machinelevel/sp425-crazy-cow#4]
May be a simple feature that could be added is the possibility to use crazy-cow in the landscape mode (and the text then being in the right direction)
Hope this is possible and easy to implement ?
Not so much needed I think but still, can be usefull, especially if someone want to build a case that would make it possible to have the rM tablet standing in front of you while you type down: because the usb connector is at the bottom, portrait mode isn't suited for that while landscape mode make it possible to plug the bottom side while the right side is actually lying on the floor.
Thanks !

**Comment by machinelevel** [comment]
Yep, this is a really good idea, and would be easy to implement. Most likely, the quickest path to getting it done would be to use an F1-F10 key or else ctrl-something to switch portrait and landscape typing.
